appengine/gulp/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "appengine-gulpjs",
+  "description": "An example of running Gulp.js on Google App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": "~4.2"
+  },
+  "scripts": {
+    "start": "node ./src/bin/www",
+    "postinstall": "gulp build",
+    "deploy": "gcloud preview app deploy"
+  },
+  "dependencies": {
+    "body-parser": "^1.14.2",
+    "cookie-parser": "^1.4.1",
+    "debug": "^2.2.0",
+    "express": "^4.13.4",
+    "jade": "^1.11.0",
+    "morgan": "^1.6.1",
+    "serve-favicon": "^2.3.0"
+  },
+  "devDependencies": {
```

Comment:
Can you move these into `dependencies`? Right now `devDependencies` won't get installed on deploy.